public/index.html

```diff
@@ -76,12 +76,12 @@ <h1>
 			const output = document.getElementById('output');
 
 			/**
-			 * Figures out if a given url is a webhook
+			 * Figures out if a given url is a valid Discord webhook
 			 * @param url {string} The webhook URL
-			 * @return url is `https://discord.com/api/webhooks/${string}/${string}`
+			 * @return a boolean indicating if the url is `https://discord.com/api/webhooks/${string}/${string}`, `https://canary.discord.com/api/webhooks/${string}/${string}`, or `https://ptb.discord.com/api/webhooks/${string}/${string}`
 			 */
 			function isWebhook(url) {
-				return url.startsWith('https://discord.com/api/webhooks/');
+				return ["https://discord.com", "https://canary.discord.com", "https://ptb.discord.com"].some(domain => url.startsWith(domain) && url.includes('/api/webhooks'));
```

Comment:
Consider using a regex here, to allow for future subdomains down the line along with support for `www.`, & `http://`

Reply:
Yep, a regex would be fantastic if possible!